core/prelude-core.el

```diff
@@ -285,6 +285,24 @@ there's a region, all lines that region covers will be duplicated."
   (interactive)
   (byte-recompile-directory prelude-dir 0))
 
+(defun prelude-file-uid (filename)
+  "Return the UID of the FILENAME, normally as a string.
+
+See `file-attributes for more info.'"
+  (nth 2 (file-attributes filename)))
+
+(defun prelude-file-owned-by-user-p (filename)
+  "Return t if file FILENAME is owned by the currently logged in user."
+  (let ((uid (prelude-file-uid filename)))
+    (or (eq uid
```

Comment:
I just read the docs again and there's an even better way.  You can specify their return values with the optional third argument.  So I'll specify `'integer` and then only do one check.